<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>221</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>221</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>221</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="buttonPers">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="buttonClear">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>СЕ</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="buttonClAll">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>С</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="buttonAdd">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="buttonNum1">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buttonNum2">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="buttonNum3">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="buttonSub">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="buttonNum4">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="buttonNum5">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="buttonNum6">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="buttonMult">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="buttonNum7">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="buttonNum8">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="buttonNum9">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="buttonDiv">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="buttonSign">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>±</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="buttonNum0">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="buttonDot">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="buttonEqu">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonNum0</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>numKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNum1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>numKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNum2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>numKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNum3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>numKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNum4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>numKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNum5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>numKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNum6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>numKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNum7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>numKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNum8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>numKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNum9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>numKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mathOperKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonMult</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mathOperKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDiv</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mathOperKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonPers</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mathOperKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSub</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mathOperKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClear</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClAll</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSign</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeSignKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonEqu</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resultKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDot</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>dotKeyPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
